test/patterns/classic_setter/test_classic_setter.py

```diff
@@ -43,11 +43,14 @@ def test_four_setter_patterns(self):
         ast = AST.build_from_javalang(build_ast(filepath))
         pattern = ClassicSetter()
         lines = pattern.value(ast)
-        self.assertEqual(lines, [1236, 1240, 3783, 3819])
+        self.assertEqual(lines,
```

Comment:
Why here and in next test values changed?

Reply:
I have explained it in the description:
 Pattern checked whether function name equals the filed name the following way:
field bitMap, function setBitMap ->setter

But now i just check if only the function begins with `set`

Is it clearer now?